providers/linux/capabilities_linux.go

```diff
@@ -83,40 +83,36 @@ func capabilityName(num int) string {
 	return strconv.Itoa(num)
 }
 
-func readCapabilities(content []byte) (*types.CapabilityInfo, error) {
-	var cap types.CapabilityInfo
-
-	err := parseKeyValue(content, ':', func(key, value []byte) error {
+func decodeCapabilityLine(content string, capInfo *types.CapabilityInfo) error {
+	return parseKeyValue([]byte(content), ':', func(key, value []byte) error {
 		var err error
 		switch string(key) {
 		case "CapInh":
```

Comment:
although this code was already like that, I debate inside me if it would add more value having all Cap* strings defined as constants. Feel free to ignore

Reply:
Ack, that makes sense, I was aiming to be conservative and using what's already there though.